src/main/java/org/isf/medicalinventory/gui/InventoryBrowser.java

```diff
@@ -267,7 +267,6 @@ private JPanel getPanelFooter() {
 			panelFooter.add(getNewButton());
 			panelFooter.add(getUpdateButton());
 			panelFooter.add(getViewButton());
-			panelFooter.add(getPrintButton());
```

Comment:
You have removed the call to the `getPrintButton()` method but the method still exists in the class at line 413.

In addition the variable `jButtonPrint` defined at line 87 is still in the class.